R/agglomerate.R

```diff
@@ -242,8 +252,9 @@ setGeneric("agglomerateByVariable",
 #'
 #' @export
 setMethod("agglomerateByRank", signature = c(x = "SummarizedExperiment"),
-    function(x, rank = taxonomyRanks(x)[1], onRankOnly = TRUE, na.rm = FALSE,
-        empty.fields = c(NA, "", " ", "\t", "-", "_"), ...){
+    function(x, rank = taxonomyRanks(x)[1], onRankOnly= TRUE, na.rm = FALSE, 
+             empty.fields = c(NA, "", " ", "\t", "-", "_"), disable.taxonomy = onRankOnly, 
```

Comment:
The name parameter could be there first, next to old. It might be hard to see. The order is same in documentation

-->

disable.taxonomy = onRankOnly, onRankOnly = TRUE

Also check that the indentation is correct in lines that you add (multiplication of 4) https://contributions.bioconductor.org/r-code.html?q=indent#indentation

Reply:
Ahh was browsing some failing tests then forgot to reorder it. its fixed now.